### PR TITLE
installed phpUnit, beacuse we gonna use it for writing test

### PR DESCRIPTION
**Pull request by dimaMikky**
check it please

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Lagoon development
 
 1. installed GrumPhP, because we gonna use it for check future commits
+2. installed PHPUnit, because we gonna use it for writing tests
```
Comment (ibudasov) [inline README.md]: NBC: is it some sort of a changelog?

File: composer.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
         "symfony/debug-bundle": "^3.3|^4.0",
         "symfony/dotenv": "^4.0",
         "symfony/monolog-bundle": "^3.0",
-        "symfony/phpunit-bridge": "^3.3|^4.0",
+        "symfony/phpunit-bridge": "^4.0",
```
Comment (ibudasov) [inline composer.json:24]: I don't see the actual phpunit/phpunit, where is it?

File: src/Controller/Calculator.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace App\Controller;
+
+class Calculator
+{
+    public function add($a, $b)
+    {
+        return $a + $b;
+    }
+}
```
Comment (ibudasov) [inline src/Controller/Calculator.php:13]: You have installed and configured GrumPHP, right? 

It should block all the code:
- without typehints
- without return types
- no empty line at the end
- with commented out code
- no declare('strict_types=1');

All those things are bad and must be filtered out by robots.
Ping me if you need help with configuring it.

Also add all of this to the codestyle, please.

File: tests/Controller/CalculatorTest.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<?php
+// tests/Controller/CalculatorTest.php
+namespace App\Tests\Util;
+
+use App\Controller\Calculator;
+use PHPUnit\Framework\TestCase;
+
+class CalculatorTest extends TestCase
```
Comment (ibudasov) [inline tests/Controller/CalculatorTest.php:8]: Look, PHPUnit is actually here!

File: tests/Controller/CalculatorTest.php [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public function testAdd()
```
Comment (ibudasov) [inline tests/Controller/CalculatorTest.php:10]: Let's make the name more explanatory!

File: tests/Controller/CalculatorTest.php [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        $calculator = new Calculator();
+        $result = $calculator->add(30, 12);
+
+        // assert that your calculator added the numbers correctly!
```
Comment (ibudasov) [inline tests/Controller/CalculatorTest.php:15]: You want to put such a comment in the assertion as a 3rd parameter

**Comment by DmytroHudzenko** [comment]
> check it please

i think it is great code . What do you think about it @ibudasov ?

**Comment by dimaMikky** [comment]
> still waiting the response

**Review by ibudasov — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (dimaMikky) [reply composer.json:24]: According the symfony testing guide i use the phpunit-bridge. It wraps the original PHPUnit binary to provide additional features.

File: README.md (hunk 1 above)
Comment (dimaMikky) [reply README.md]: good mention, ill fix it

Comment (ibudasov) [reply README.md]: nono I mean, if you want it here - np, I'm just asking what is it for?

File: composer.json (hunk 2 above)
Comment (ibudasov) [reply composer.json:24]: oh ok, let it be then

File: README.md (hunk 1 above)
Comment (new2lagoon) [reply README.md]: i put some changes, can you see them ?

Comment (dimaMikky) [reply README.md]: @ibudasov , i put some changes. Can you see it ?

Comment (ibudasov) [reply README.md]: yep, I do, but I don't see the PR in the list of PRs to review anymore :(

**Pull request closed without merging**